src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializableV1.java

```diff
@@ -171,6 +171,7 @@ public String toString()
   List<MetaDataOfBinds> metaDataOfBinds = new ArrayList<>();
   QueryResultFormat queryResultFormat;
   boolean treatNTZAsUTC;
+  SFSession session;
```

Comment:
I think session is not serializable object. I think we'd better not add it. if we really need to add it, please mark it as "transient" after "transient TimeZone timeZone;"

I think our test case should raise error for this.

Reply:
In addition, if a "transient" field is added, think over to initialize it in setupTransientFields() which is used to setup transient fields when creating result set without connection.